### PR TITLE
Shows progress on the participant view, other minor styling changes

### PR DESCRIPTION
**Pull request by ruiramos**
This PR adds progress indication on the participant view through a progress bar and a count (number of questions/total).

It also tries to bring some styles together like headings, header and loading text.

### CODE REVIEW

File: pages/jams/[jam].js [hunk 1]
```diff
@@ -175,32 +183,41 @@ const Jam = () => {
...
-          : ''}
+        {question && question.isUserSubmitted && (
+          <GridItem colSpan={4}>
+            <HStack>
+              <InfoOutlineIcon />
+              <Text fontSize="xs">
+                <strong>Participant</strong> submitted{' '}
+                {moment(question.createdAt?._seconds * 1000).format(
+                  'DD MMM hh:mm A',
+                )}
```
Comment (siame) [inline pages/jams/[jam].js:215]: You can use `convertDate` from `utils/date.js` to do this so we keep things consistent.

File: pages/jams/[jam].js [hunk 2]
```diff
@@ -175,32 +183,41 @@ const Jam = () => {
...
       />
+      {question && (
+        <Progress
+          h="0.5"
+          value={
+            (question.meta.numVotes / question.meta.numQuestions) *
+            100
+          }
+        />
+      )}
```
Comment (siame) [inline pages/jams/[jam].js:194]: The progress bar currently disappears when we complete the Jam (as there's no more `question` object), any way we can show it as complete?

File: pages/jams/[jam].js (hunk 1 above)
Comment (ruiramos) [reply pages/jams/[jam].js:215]: Thanks will update

File: pages/jams/[jam].js (hunk 2 above)
Comment (ruiramos) [reply pages/jams/[jam].js:194]: We can for sure!

**Review by siame — APPROVED** [review]

**Review by Midnite — APPROVED** [review]

**Pull request merged**